app/api/v1/auth/index.js

```diff
@@ -175,4 +212,25 @@ router.post('/verification', (req, res, next) => {
   });
 });
 
+/**
+ * Verifies a user's email based on a code sent out to the user's provided email
+ * Responds with a JSON object with fields 'error', 'verified' (if a user is succesfully verified)
+ */
+router.post('/verifyEmail', (req, res, next) => {
```

Comment:
Use the same `emailVerification` route and match password reset:

`GET /resetPassword/:email` + `POST /resetPassword/:accessCode` &rarr;
`GET /emailVerification/:email` + `POST /emailVerification/:accessCode`